intro-OpenStack/ch_ManageImages.tex

```diff
@@ -0,0 +1,182 @@
+\chapter{Upload and manage images}
+
+A virtual machine \gls{OpenStack Image}, referred to in this document simply as an
+image, is a single file that contains a virtual disk that has a bootable
+operating system installed on it. Images are used to create virtual
+machine instances within the cloud. For information about creating image
+files, see the
+\href{https://docs.openstack.org/image-guide/}{\emph{OpenStack Virtual
+Machine Image Guide}}.
+
+Depending on your role, you may have permission to upload and manage
```

Comment:
Do VSC cloud users have permission to upload and manage their own images, or not? If we know, we can avoid the "depending on [...] you may have permission"